Core/src/main/java/com/paypal/android/core/graphql/fundingEligibility/enums.kt

```diff
@@ -0,0 +1,13 @@
+package com.paypal.android.core.graphql.fundingEligibility
+
+enum class SupportedPaymentMethodsType {
```

Comment:
Can we break each of these out into their own files to improve discoverability for newly onboarded contributors?

Reply:
Yeah, definitely!